packages/block-editor/src/checkbox-input/index.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { CheckboxInput } from '@crowdsignal/blocks';
+import { CheckboxInputIcon } from '@crowdsignal/icons';
+import attributes from './attributes';
+import EditFreeText from './edit';
+
+const settings = {
+	apiVersion: 1,
+	title: __( 'Single Checkbox', 'block-editor' ),
+	description: __(
+		'Do you need consent to something like terms of service?',
+		'block-editor'
+	),
+	category: 'crowdsignal-forms/form',
+	keywords: [
```

Comment:
These are the search keywords, it looks like the original stuff for the text input box is still here so the checkbox will show up if a user searches for free text and stuff like that.  But we can also use it for other things users might search for like "terms of service"

Reply:
Removed unnecessary keywords and added 'terms of service'